packages/psss-server/src/routes/reports/ConfirmCountryWeeklyReportRoute.ts

```diff
@@ -37,12 +56,93 @@ export class ConfirmCountryWeeklyReportRoute extends Route {
     const answers = mapUnconfirmedReportToConfirmedAnswers(report.results[0]);
 
     return this.meditrakConnection?.updateOrCreateSurveyResponse(
-      SURVEY_CODE,
+      CONFIRMED_WEEKLY_SURVEY_COUNTRY,
       countryCode,
       week,
       answers,
     );
   }
+
+  async createAlerts(countryCode: string, week: string) {
```

Comment:
Love to see this renamed to something like `updateAlerts()` as we are both creating and arching existing alerts. Bonus points if you refactor out the internal logic of creating an alert or archiving one so it can read something like:
```
updateAlerts() {
  
  // ... setup ...

  if (!syndromeAlerts && result[`${syndromeCode} Threshold Crossed`] === true) {
    const newAlert = createAlert();
    response.createdAlerts.push(newAlert);
  }
  
  if (currentWeekSyndromeAlert && result[`${syndromeCode} Threshold Crossed`] === false) {
    archiveAlert();
  }
}
```